src/SUMMARY.md

```diff
@@ -231,10 +231,10 @@
 
 - [Async](async.md)
   - [async/await](async/async-await.md)
-  - [Async Blocks](async/async-blocks.md)
   - [Futures](async/futures.md)
   - [Runtimes](async/runtimes.md)
-  - [Tasks](async/tasks.md)ures](async/futures.md)
+    - [Spawning](async/runtimes/spawning.md)
+    - [Tasks](async/runtimes/tasks.md)
```

Comment:
Renamed `spawning` to `tokio` as it serves as an intro to `tokio`. But essentially it introduces spawning too.

Also moved `Tasks` back to top-level. I think `async/runtimes/tasks.md` can be simplified, and perhaps the example in it could be moved to a separate `example` slide, since it the students have to run it on their local machines.

Reply:
Sounds good

Reply:
We might want to keep it to get deeper into spawning. I need to have a closer look, after merging this.